### PR TITLE
Implement DecodingLayer interface for loopback

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

**Comment by gconnell** [comment]
Great stuff, thanks!  Could you do me a quick favor and add godoc comments to the functions?  They'll be super redundant to everything else, but they make the godoc pages look a little nicer :)

**Comment by urso** [comment]
I signed it!

File: layers/loopback.go [hunk 1]
```diff
@@ -23,6 +23,33 @@ type Loopback struct {
...
+}
+func (l *Loopback) CanDecode() gopacket.LayerClass {
+	return LayerTypeLoopback
+}
+
+func (l *Loopback) NextLayerType() gopacket.LayerType {
+	return l.Family.LayerType()
+}
+
 func decodeLoopback(data []byte, p gopacket.PacketBuilder) error {
```
Comment (gconnell) [inline layers/loopback.go]: Switch decodeLoopback to use DecodeFromBytes?

**Comment by urso** [comment]
Added godoc comments

**Comment by urso** [comment]
ping

**Pull request merged**